test/controllers/organization_memberships_controller_test.exs

```diff
@@ -23,6 +23,8 @@ defmodule CodeCorps.OrganizationMembershipControllerTest do
     }
   end
 
+  defp get_role(data), do: data["attributes"]["role"]
```

Comment:
Revised in https://github.com/code-corps/code-corps-api/pull/329/commits/120f71d6be94a9955b8e526b7e4bba7bc33d9af6